src/dj/auth/views.py

```diff
@@ -0,0 +1,25 @@
+from django.shortcuts import render
+from django.views import View
+from django.views.decorators.csrf import csrf_exempt
+from django.utils.decorators import method_decorator
+from django.http import JsonResponse
+
+
+@method_decorator(csrf_exempt, name="dispatch")
+class LoginView(View):
+    def post(self, request, *args, **kwargs):
+        assert False, request
+
+
+class GetIdentity(View):
+    def get(self, request, *args, **kwargs):
+        session_id = request.GET.get("sid") or None
```

Comment:
```suggestion
        session_id = request.GET.get("sid")
```

Is equivalent.

Reply:
or None also sets it to None if sid is an empty string. Otherwise we'll have to make a check for empty string as well.